common/app/common/CSV.scala

```diff
@@ -24,23 +25,23 @@ object CSV extends implicits.Strings {
 
     def COMMA = ","
     def DQUOTE = "\""
-    def DQUOTE2 = "\"\"" ^^ { case _ => "\"" } // combine 2 dquotes into 1
-    def CRLF = "\r\n" | "\n"
-    def TXT = "[^\",\r\n]".r
-    def SPACES = "[ \t]+".r
+    def DQUOTE2: _root_.common.CSV.Parser.Parser[String] = "\"\"" ^^ (_ => "\"")
+
+    // combine 2 dquotes into 1
+    def CRLF: _root_.common.CSV.Parser.Parser[String] = "\r\n" | "\n"
```

Comment:
Is it possible to `import _root_.common.CSV.Parser.Parser[String]`. Then it would become `def CRLF: Parser[String] = "\r\n" | "\n"` which is much more readable